src/broker/overlay.c

```diff
@@ -963,7 +963,9 @@ static void parent_cb (flux_reactor_t *r, flux_watcher_t *w,
                 logdrop (ov, OVERLAY_UPSTREAM, msg, "malformed keepalive");
             }
             else if (ktype == KEEPALIVE_DISCONNECT) {
-                flux_log (ov->h, LOG_ERR, "parent disconnect");
+                log_msg ("%s (rank %lu) sent disconnect control message",
```

Comment:
I was under the impression that `flux_log (h, LOG_ERR, ..)` is already sent to stderr if the severity is <= `log-stderr-level`?

Reply:
Well not currently.  Maybe we should consider switching this default to `local`
```
log-stderr-mode
   If set to "leader" (default), broker rank 0 emits forwarded logs from
   other ranks to stderr, subject to the constraints of log-forward-level
   and log-stderr-level.  If set to "local", each broker emits its own
   logs to stderr, subject to the constraints of log-stderr-level.
```

Reply:
Actually there is also this:
```
log-critical-level
   Log entries at syslog(3) level at or below this value are copied
   to stderr on the logging rank, for capture by the enclosing instance.
```
Which currently defaults to LOG_CRIT.  I could log this message at LOG_CRIT.

Perhaps in a future PR we can adjust log-critical-level and log-stderr-mode.

Reply:
Sounds good to me! Thanks for the clarification.

Something to consider: for all single user instances stderr of rank 0 is _probably_ the same destination as all other ranks, since we're running in a parallel job where stderr is captured from all job ranks, or we're running with `flux start -s` where `stderr` is shared.

In the system instance case, stderr is captured in the local systemd journal, so it would probably be really helpful if local `LOG_ERR` and higher severity error messages were captured here by default because that might be the first place a sysadmin would look.

(Not sure if that actually clarifies anything, except that copying error messages to stderr might be best for most circumstances, unless it causes the messages to be duplicated at rank0)

In any event, nothing needs changing in this PR afaics.

Reply:
Thanks!  I opened that as a separate issue.